packages/start/src/client/createServerFn.ts

```diff
@@ -308,11 +308,14 @@ const applyMiddleware = (
 function execValidator(validator: AnyValidator, input: unknown): unknown {
   if (validator == null) return {}
 
-  if ('~validate' in validator) {
-    const result = validator['~validate']({ value: input })
+  if ('~standard' in validator) {
+    const result = validator['~standard'].validate(input)
 
     if ('value' in result) return result.value
 
+    if (result instanceof Promise)
+      throw new Error('Async validation not supported')
+
```

Comment:
I also fixed an existing typing error that was blocking the build process. @tannerlinsley I updated this to be consistent with the router implementation. Or was using `~validate` intentional?